docs/deploy_features.md

```diff
@@ -16,8 +16,27 @@ You should typically enable these features before running `azd up`. Once you've
 
 We generally find that most developers are able to get high quality answers using GPT 3.5. However, if you want to try GPT-4, you can do so by following these steps:
 
-* In `infra/main.bicep`, change `chatGptModelName` to 'gpt-4' instead of 'gpt-35-turbo'.
-* You may also need to adjust the capacity above that line depending on how much TPM your account is allowed.
+1. Execute the following command inside your terminal to use GPT-4:
+
+    ```bash
+    azd env set USE_GPT4 true
+    ```
+
+2. Execute the following command inside your terminal to set the name of the GPT-4 deployment:
+
+    ```bash
+    azd env set AZURE_OPENAI_CHATGPT_DEPLOYMENT chat4
+    ```
+
+You may also need to adjust the capacity in `infra/main.bicep` depending on how much TPM your account is allowed.
+
+> [!NOTE]
+> To revert back to GPT 3.5, run the following commands:
+> - `azd env set USE_GPT4 false` to disable GPT-4.
+> - `azd env set AZURE_OPENAI_CHATGPT_DEPLOYMENT <YOUR-DEPLOYMENT-NAME>` to use the name for your GPT-3 deployment.
```

Comment:
@pamelafox you also need to ask people to set the name if they want to revert back to a previous thing as it will try to use the old name when they just disable the feature without going back to the old name or adding a new name.